src/main/scala/strawman/collection/mutable/Builder.scala

```diff
@@ -1,27 +1,27 @@
 package strawman.collection.mutable
 
-import scala.{Boolean, Any, Char}
+import scala.{Boolean, Any, Char, Unit}
 import java.lang.String
 import strawman.collection.{IterableMonoTransforms, IterableOnce}
 
 /** Base trait for collection builders */
-trait Builder[-A, +To] { self =>
+trait Builder[-A, +To] extends Growable[A] { self =>
 
   /** Append an element */
   def +=(x: A): this.type
 
+  /** Clears the contents of this builder.
+   *  After execution of this method the builder will contain no elements.
+   */
+  def clear(): Unit
```

Comment:
Note that the same holds for `+=`. It's also added in `Builder` for documentation.